### DISCUSSION TITLE
How is RDF-Star support accomplished?

### DISCUSSION THREAD

**Discussion by lambdakris** [morph-kgc/morph-kgc#109] [Q&A] [answered]
Hi, just a simple question. I was curious how the support for RDF-Star is accomplished since to the best of my knowledge, neither rdflib nor pyoxigraph have support for RDF-Star.

**Comment by arenas-guerrero-julian** [comment, marked as answer]
Hi @lambdakris ,

Indeed, RDFlib does not support RDF-star, but pyoxigraph does (since version  [0.3.0](https://github.com/oxigraph/oxigraph/releases/tag/v0.3.0)).

Also, RDF-star is supported when executing Morph-KGC via command line since RDFlib and Oxigraph are not needed.

Julián

**Comment by lambdakris** [comment]
Ah excellent! Thank you for the answer @ArenasGuerreroJulian. Much appreciated.

**Comment by arenas-guerrero-julian** [comment]
This is better explained in our [SWJ paper](https://www.semantic-web-journal.net/system/files/swj3238.pdf) (under review).
